getting_started/README.md

```diff
@@ -0,0 +1,74 @@
+# Getting started
+
+Hi and welcome to Crystal's Reference Book!
+
+First we are going to check if we have [installed the language](https://crystal-lang.org/install/) correctly so that we may try all the examples listed in this book.
+
+Once installed, the Crystal compiler should be available as `crystal` command.
+
+Let's try it!
+
+## Crystal version
+
+We may check the Crystal compiler version. If Crystal is installed correctly then we should see something like this:
+
+```terminal-session
+$ crystal --version
+Crystal 0.33.0 (2020-02-14)
```

Comment:
This is already outdated at print time 😄

Reply:
🙈